localstack/services/apigateway/provider.py

```diff
@@ -638,12 +641,56 @@ def untag_resource(
         for key in tag_keys:
             resource_tags.pop(key, None)
 
+    def import_rest_api(
+        self,
+        context: RequestContext,
+        body: Blob,
+        fail_on_warnings: Boolean = None,
+        parameters: MapOfStringToString = None,
+    ) -> RestApi:
+
+        openapi_spec = parse_json_or_yaml(to_str(body))
+        response = _call_moto(
+            context,
+            "CreateRestApi",
+            CreateRestApiRequest(name=openapi_spec.get("info").get("title")),
+        )
+
+        return _call_moto(
+            context,
+            "PutRestApi",
+            PutRestApiRequest(
+                restApiId=response.get("id"),
+                failOnWarnings=str_to_bool(fail_on_warnings) or False,
+                parameters=parameters or {},
+                body=body,
+            ),
+        )
+
 
 # ---------------
 # UTIL FUNCTIONS
 # ---------------
 
 
+def _call_moto(context: RequestContext, operation_name: str, parameters: ServiceRequest):
+    """
+    Not necessarily the pattern we want to follow in the future, but this makes possible to nest
+    moto call and still be interface compatible.
+
+    Ripped :call_moto_with_request: from moto.py but applicable to any operation (operation_name).
+    """
+    local_context = create_aws_request_context(
```

Comment:
So this was result of a conversation with @thrau, we don't necessarilly think this will be the pattern to use, hence kept it close where it's used. Moving it to the `moto.py` is not problem for me, but might result in being spread unnecessarily?